src/codec/framers/character_delimited.rs

```diff
@@ -0,0 +1,317 @@
+use crate::codec::{BoxedFramer, BoxedFramingError, FramingConfig};
+use bytes::{Buf, BufMut, Bytes, BytesMut};
+use serde::{Deserialize, Serialize};
+use std::{cmp, io, usize};
+use tokio_util::codec::{Decoder, Encoder};
+
+/// Config used to build a `CharacterDelimitedCodec`.
+#[derive(Debug, Clone, Default, Deserialize, Serialize)]
+pub struct CharacterDelimitedDecoderConfig {
+    /// The character that delimits byte sequences.
+    delimiter: char,
+    /// The maximum length of the byte buffer.
+    max_length: Option<usize>,
```

Comment:
It should be made explicit that (as far as I can tell from the code below) this length does *not* include the trailing delimiter.

Reply:
Added in 935256e0787542222b664cdba1cb4aa4b3f31d7f